packages/frontend/src/utils/createSlateNode.tsx

```diff
@@ -0,0 +1,78 @@
+import { Colors } from '../constants/Colors';
+import { ElementType, ILinkElement } from './types';
+
+export const createParagraphNode = () => ({
+  type: ElementType.PARAGRAPH_TWO,
+  children: [{ text: '' }],
+});
+
+export const createLinkNode = (href: string, text: string): ILinkElement => ({
+  type: ElementType.LINK,
+  href,
+  children: [{ text }],
+});
+
+export const createNotificationNode = (notificationType: string) => ({
+  type: notificationType || ElementType.NOTIFICATION_INFO,
+  children: [{ text: '' }],
+});
+
+export const createContactCardNode = () => ({
+  type: ElementType.CARD,
+  children: [
+    {
+      children: [{ type: ElementType.CARD_TITLE, children: [{ text: 'Yhteystiedot' }] }],
+    },
+    {
+      children: [{ text: '' }],
+    },
+    {
+      children: [{ text: 'Vastuullisuus 1', bold: true }],
```

Comment:
These texts should be in translations (except for that test1@example.com).  What is the English word here? Vastuullisuus sounds a bit off. Vastuu might be better, but depends on what we're looking for here.